managed/services/supervisord/supervisord.go

```diff
@@ -48,6 +48,7 @@ import (
 
 const (
 	defaultClickhouseDatabase       = "pmm"
+	defaultHostAddress              = "localhost"
```

Comment:
It seems like this should be 127.0.0.1, since `localhost` could, in theory, map to anything through `/etc/hosts`.

Reply:
Also, our nginx.conf operates mostly with `127.0.0.1`.